lib/gollum-lib/wiki.rb

```diff
@@ -162,25 +162,13 @@ def exist?
 
     # Public: Get the formatted page for a given page name, version, and dir.
     #
-    # name    - The human or canonical String page name of the wiki page.
+    # path    - The String path to the the wiki page (may or may not include file extension).
     # version - The String version ID to find (default: @ref).
-    # dir     - The directory String relative to the repo.
     #
     # Returns a Gollum::Page or nil if no matching page was found.
-    def page(name, version = @ref, dir = nil, exact = false)
+    def page(path, version = @ref)
```

Comment:
If `@ref` always set?

Reply:
Or can `version` be `nil`?

Reply:
the `nil` case is handled on the next line